src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/prison/PrisonApiClient.kt

```
@@ -61,29 +47,18 @@ class PrisonApiClient(@Qualifier("oauthPrisonClient") val prisonerApiWebClient:
       .accept(MediaType.APPLICATION_JSON)
       .retrieve()
       .bodyToMono(typeReference<List<CourtEventOutcome>>())
-      .onErrorResume { webClientErrorHandler(it) }
       .block() ?: emptyList()
   }
 
-  private fun <API_RESPONSE_BODY_TYPE> webClientErrorHandler(exception: Throwable): Mono<API_RESPONSE_BODY_TYPE> =
+  private fun <API_RESPONSE_BODY_TYPE> coerce404ResponseToNull(exception: Throwable): Mono<API_RESPONSE_BODY_TYPE> =
     with(exception) {
-      if (this is WebClientResponseException) {
-        val uriPath = request?.uri?.path
-        when (statusCode) {
-          FORBIDDEN -> {
```

This should be treat as an unexpected error and fail LOUDLY